app/styles.css

```
@@ -248,6 +236,15 @@
 }
 
 @media (max-width: 767px) {
+  [data-slot="dialog-content"] {
+    --dialog-safe-top: max(1rem, calc(env(safe-area-inset-top) + 0.75rem));
+    --dialog-safe-bottom: max(1rem, calc(env(safe-area-inset-bottom) + 0.75rem));
+    top: var(--dialog-safe-top);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)